packages/teleport/src/SideNav/SideNav.tsx

```diff
@@ -94,3 +99,5 @@ export const Content = styled.div`
   flex-direction: column;
   overflow: auto;
 `;
+
+const isExternalLink = (link: string): boolean => link.startsWith('http');
```

Comment:
what do you think about putting this here: https://github.com/gravitational/webapps/blob/4da005d774cff0820b50439541a3d6ee1f89d0b3/packages/teleport/src/SideNav/useSideNav.ts#L102, as an optional `externalLink?: boolean` field?

Reply:
I think this is better. This way we explicitly define that a nav item is external in `features`, rather than checking for `http`. Added in https://github.com/gravitational/webapps/pull/1452/commits/a2341e7016700db8c9afb309fdf24bb126b1c49f